src/geo/map-view.js

```diff
@@ -154,20 +154,18 @@ var MapView = View.extend({
     this._unbindModel();
     this.map.bind('change:view_bounds_sw', this._changeBounds, this);
     this.map.bind('change:view_bounds_ne', this._changeBounds, this);
-    this.map.bind('change:zoom', this._setZoom, this);
```

Comment:
Why not have:

```
this.map.bind('change:zoom', this._setZoom, this);
this.map.bind('change:center', this._setCenter, this);
```

?

Perhaps we can even get rid of _setView which is a bit confusing?